bps/clustering/DRP-recalibrated.yaml

```diff
@@ -0,0 +1,58 @@
+# This is a prescription for quantum clustering with BPS, suitable for any
+# concrete pipeline based on the "DRP-recalibrated" pipeline.  It also includes
+# clustering for analysis_drp plots.
+#
+# Use it by adding:
+#
+#   includeConfigs:
+#    - ${DRP_PIPE_DIR}/bps/clustering/DRP-recalibrated.yaml
```

Comment:
Should this line be indented by one more whitespace? I.e., so that the `-` is directly below the `c` in `includeConfigs`?